docs/products/kafka/howto/approvals.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Manage approvals
+limited: true
+---
+
+The approvals page allows you to manage requests for Aiven for Apache Kafka® resources owned by your group.
+These requests can include claims for existing topics or requests to create new topics.
+
+## Requests flow
+
+Managing requests involves several key steps:
+
+- **Request initiation**: Users submit a request from the Apache Kafka topic catalog to
+  claim ownership of an existing topic or create a topic.
+- **Request visibility**: All members of the topic's owning group can see the request on
+  the approvals page.
+- **Review and approval**: A group member reviews the request, and either approves or
+  declines it.
+- **Completion**: If approved, the requested topic ownership change or topic creation
+  is completed. If declined, the request is not implemented.
+
+## Key elements
+
+The approvals page displays the following key elements:
+
+- **Topic**: Name of the topic
+- **Type**: Claim or create request
+- **Status**: Current status of the request
+  - **Approved**: Request approved and processed
+  - **Declined**: Request declined. The topic is now available for another group to claim
```

Comment:
```suggestion
  - **Declined**: Request declined. The topic is now available for another group to claim.
```

Reply:
I would skip the period here as per Google styleguide